cvss/tests/uncovered-v4.rs

```diff
@@ -0,0 +1,20 @@
+#![cfg(all(feature = "v4", feature = "std"))]
+/// Regression Tests
+///
+/// NOTE: These CVEs are actually `CVSS:4.0` and have been picked from real
+/// practical cases not covered by Red Hat
```

Comment:
These should probably be toplevel rustdoc comments, and I'm used to the attributes going underneath those.
```suggestion
//! Regression Tests
//!
//! NOTE: These CVEs are actually `CVSS:4.0` and have been picked from real
//! practical cases not covered by Red Hat

#![cfg(all(feature = "v4", feature = "std"))]
```